frontend/src/metabase-lib/filter.ts

```diff
@@ -758,7 +766,7 @@ function relativeDateFilterPartsWithOffset(
     column,
     value: startValue < 0 ? startValue : endValue,
     bucket: startBucket,
-    offsetValue,
+    offsetValue: offsetValue * -1,
```

Comment:
Bug 4 - we negate the offset value when constructing an expression but don't do it here (tested) https://github.com/metabase/metabase/blob/9abaf80bfb4bb9a53e02959de2cb95419953ef93/frontend/src/metabase-lib/filter.ts#L339